packages/py-api/py_api/controllers/getting_logs_controller.py

```diff
@@ -0,0 +1,44 @@
+import pathlib
+from os.path import exists
+from traceback import format_exc
+from typing import Final
+
+from fastapi import APIRouter
+from py_api.environment import IS_OFFLINE
+from starlette.responses import FileResponse, JSONResponse
+
+router = APIRouter()
+LOGFILE_PATH: Final = f"{pathlib.Path(__file__).parent.resolve().parent}/shared/logfile.log"
+
+
+class GettingLogsController:
+    @classmethod
+    def return_files(cls) -> JSONResponse | FileResponse:

```

Comment:
Why do you have two methods `get_log_file` and `get_files`. 
You should learn to call the functions by what they are actually doing. 

I expect to load a handler for a file or get a blob/bytes array when calling `get_log_file`, not a JSONResponse or a FileResponse.

Reply:
No need for two separate methods here. 
Combine them and drop the try catch clause.
Check whether the file exists and that we're in the correct environment in the beginning of the method. 

Return appropriate response, no need for handling other exceptions than being in the incorrect environment and logs not being present.